### ISSUE TITLE
DensifyPointCloud: error while loading shared libraries: libcuda.so.1

### ISSUE THREAD

**Issue by sean-kenops** [OpenDroneMap/NodeODM#216]
### What is the problem?

DensifyPointCloud: error while loading shared libraries: libcuda.so.1

### What should be the expected behavior?

libcuda.so.1 should be found in the docker image.

Looks like the expected path is missing in the image. Adding this path corrects the issue.

mkdir /usr/local/nvidia
ln -s  /usr/local/cuda-11.2/compat /usr/local/nvidia/lib
ls -l /usr/local/nvidia/lib/

### How can we reproduce this? (What steps did you do to trigger the problem? Be detailed)

Further details are located in the forum: https://community.opendronemap.org/t/densifypointcloud-error-while-loading-shared-libraries-libcuda-so-1/20178/1

[INFO]    Running dense reconstruction. This might take a while.
[INFO]    Estimating depthmaps
[INFO]    running "/code/SuperBuild/install/bin/OpenMVS/DensifyPointCloud" "/var/www/data/254e5ea9-97b8-4129-a552-56a9218aed51/submodels/submodel_0000/opensfm/undistorted/openmvs/scene.mvs" --resolution-level 2 --dense-config-file "/var/www/data/254e5ea9-97b8-4129-a552-56a9218aed51/submodels/submodel_0000/opensfm/undistorted/openmvs/Densify.ini" --max-resolution 4032 --max-threads 8 --number-views-fuse 2 --sub-resolution-levels 2 --archive-type 3 -w "/var/www/data/254e5ea9-97b8-4129-a552-56a9218aed51/submodels/submodel_0000/opensfm/undistorted/openmvs/depthmaps" -v 0 --cuda-device -2
/code/SuperBuild/install/bin/OpenMVS/DensifyPointCloud: error while loading shared libraries: libcuda.so.1: cannot open shared object file: No such file or directory

**Comment by sean-kenops** [comment]
This may be from a poorly formed LD_LIBRARY_PATH variable in the container.

**Comment by pierotofy** [comment]
It should have been fixed. Update and report back if it's still an issue? 🙏

**Comment by pierotofy** [comment]
How did you launch the container?

**Comment by pierotofy** [comment]
Right; if you don't pass `--gpus all` the CUDA device and libraries will not be mapped to the container and you'll get the error reported above. 

Try with `--gpus all` and feel free to re-open if the issue persists 🙏

**Comment by pierotofy** [comment]
That's not a valid docker run command? The args to NodeODM have to be passed after opendronemap/nodeodm:gpu.
